<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loadPathButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="installPathEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="bgImageLabel">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>921</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextStepButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>510</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>下一步</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16</height>
     </size>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>已下载: </string>
    </property>
   </widget>
   <zorder>bgImageLabel</zorder>
   <zorder>installPathEdit</zorder>
   <zorder>loadPathButton</zorder>
   <zorder>nextStepButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>progressLabel</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
